CHANGELOG.md

```diff
@@ -1,13 +1,18 @@
 # CHANGELOG
 
+### 2.2.2
+
+- Fix android time picker returning today's date instead of the given date [#115](https://github.com/react-native-community/react-native-datetimepicker/pull/115)
```

Comment:
isn't this a breaking change? Shouldn't we bump to 3.0 with this?

Reply:
I believe that is a bug fix. Why this is a breaking change? see the related issue #94

Reply:
@vonovak some argument to bump to 3.0? I would like send another version to NPM with this fix

Reply:
sorry for late answer. If someone relied on the fact that `DateTimePicker`'s `onChange`  is always called with `new Date()` which only has time or date set based on if it's a date or timepicker then it might be a problem, but I agree it's closer to being a patch :)